keyboards/keychron/k12/keymaps/via/k12_via_ansi.json

```diff
@@ -0,0 +1,315 @@
+        1
+      ],
+      [
+        "HUE_PENDULUM",
+        1
+      ],
+      [
+        "HUE_WAVE",
+        1
+      ],
+      [
+        "PIXEL_FRACTAL",
+        1
+      ],
+      [
+        "PIXEL_RAIN",
+        1
+      ],
+      [
+        "TYPING_HEATMAP",
+        1
+      ],
+      [
+        "DIGITAL_RAIN",
+        1
+      ],
+      [
+        "SOLID_REACTIVE_SIMPLE",
+        1
+      ],
+      [
+        "SOLID_REACTIVE",
+        1
+      ],
+      [
+        "SOLID_REACTIVE_WIDE",
+        1
+      ],
+      [
+        "SOLID_REACTIVE_MULTIWIDE",
+        1
+      ],
+      [
+        "SOLID_REACTIVE_CROSS",
+        1
+      ],
+      [
+        "SOLID_REACTIVE_MULTICROSS",
+        1
+      ],
+      [
+        "SOLID_REACTIVE_NEXUS",
+        1
+      ],
+      [
+        "SOLID_REACTIVE_MULTINEXUS",
+        1
+      ],
+      [
+        "SPLASH",
+        1
+      ],
+      [
+        "MULTISPLASH",
+        1
+      ],
+      [
+        "SOLID_SPLASH",
+        1
+      ],
+      [
+        "SOLID_MULTISPLASH",
+        1
+      ]
+    ],
+    "supportedLightingValues": [
+      128,
+      129,
+      131
+    ]
+  },
+  "matrix": {
+    "rows": 5,
+    "cols": 14
+  },
+  "layouts": {
+    "keymap": [
+      [
+        "0,0",
+        "0,1",
+        "0,2",
+        "0,3",
+        "0,4",
+        "0,5",
+        "0,6",
+        "0,7",
+        "0,8",
+        "0,9",
+        "0,10",
+        "0,11",
+        "0,12",
+        {
+          "w": 2
+        },
+        "0,13"
+      ],
+      [
+        {
+          "w": 1.5
+        },
+        "1,0",
+        "1,1",
+        "1,2",
+        "1,3",
+        "1,4",
+        "1,5",
+        "1,6",
+        "1,7",
+        "1,8",
+        "1,9",
+        "1,10",
+        "1,11",
+        "1,12",
+        {
+          "w": 1.5
+        },
+        "1,13"
+      ],
+      [
+        {
+          "w": 1.75
+        },
+        "2,0",
+        "2,1",
+        "2,2",
+        "2,3",
+        "2,4",
+        "2,5",
+        "2,6",
+        "2,7",
+        "2,8",
+        "2,9",
+        "2,10",
+        "2,11",
+        {
+          "w": 2.25
+        },
+        "2,13"
+      ],
+      [
+        {
+          "w": 2.25
+        },
+        "3,0",
+        "3,2",
+        "3,3",
+        "3,4",
+        "3,5",
+        "3,6",
+        "3,7",
+        "3,8",
+        "3,9",
+        "3,10",
+        "3,11",
+        {
+          "w": 2.75
+        },
+        "3,13"
+      ],
+      [
+        {
+          "w": 1.25
+        },
+        "4,0",
+        {
+          "w": 1.25
+        },
+        "4,1",
+        {
+          "w": 1.25
+        },
+        "4,2",
+        {
+          "w": 6.25
+        },
+        "4,6",
+        {
+          "w": 1.25
+        },
+        "4,10",
+        {
+          "w": 1.25
+        },
+        "4,11",
+        {
+          "w": 1.25
+        },
+        "4,12",
+        {
+          "w": 1.25
+        },
+        "4,13"
+      ]
```

Comment:
this is probably a linter at work, but it's an eyesore. Please format to something human readable